### DISCUSSION TITLE
using VS extension with MS Blend VS community

### DISCUSSION THREAD

**Discussion by MarcAir** [Dotnet-Boxed/Templates#20] [Q&A]
strangely the project is not available when starting a new project. the vs extension is installed (latest verison) cannot create a new project it does not exist.
looks like possible error due to free version. the extension is looking for company name

missing/invalid 'companyname' field in package registry

**Comment by RehanSaeed** [comment]
Which version of VS are you using? When do you get that error?

I have tested it with VS 2012, 2013 and 2015 but these are the Ultimate/Enterprise versions. I haven't tried the community edition.

**Comment by MarcAir** [comment]
Thanks Rehan, 
<b>Visual Studio Community 2015</b>

**Comment by RehanSaeed** [comment]
Installing it now.

**Comment by RehanSaeed** [comment]
Ok, I just installed VS 2015 Community Edition on a spare machine and the project template is displayed correctly under the Web section of the 'New Project' dialog. I created a project from the template without any errors.

When you say "missing/invalid 'companyname' field in package registry" what do you mean. Where did you see this error?

**Comment by RehanSaeed** [comment]
I just read the title of your issue. Are you trying to create a project in Blend? Blend is not supported. I'm not even Blend has extensions.

**Comment by MarcAir** [comment]
feel like a noob - you are right. i downloaded and installed the wrong verison. using express and it works.
thanks
